src/libs/fileDownload/index.ts

```diff
@@ -1,14 +1,12 @@
 import * as FileUtils from './FileUtils';
 import * as Link from '../actions/Link';
+import {FileDownload} from './types';
```

Comment:
```suggestion
import type {FileDownload} from './types';
```

Reply:
Does it work that we can use either `export type` or `import type`? I think that it should work in pairs.
cc @fabioh8010

Reply:
@rezkiy37 Don't need to be working in pairs, `import type` can be useful in some situations e.g. fix circular dependencies, but is not a requirement.